maxim/maxim.py

```diff
@@ -102,9 +98,8 @@ def apply(x):
 
         # Get multi-scale input images
         for i in range(1, num_supervision_scales):
-            resizing_layer = Resizing(
-                height=h // (2 ** i),
-                width=w // (2 ** i),
+            resizing_layer = ResizingDown(
+                ratio=(2**i),
                 method="nearest",
                 antialias=True,  # Following `jax.image.resize()`.
                 name=f"initial_resizing_{K.get_uid('Resizing')}",
```

Comment:
Seems like Black formatting wasn't applied.

Reply:
This is weird, to me the default behavior of black is to remove the spaces around the ** operator...

Reply:
You're right. Sorry for the false alarm.